sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/BloomFilterMightContain.scala

```diff
@@ -40,13 +40,12 @@ import org.apache.spark.util.sketch.BloomFilter
  */
 case class BloomFilterMightContain(
     bloomFilterExpression: Expression,
-    valueExpression: Expression) extends BinaryExpression {
+    valueExpression: Expression) extends BinaryExpression with Predicate {
```

Comment:
AFAK, although the `dataType` of `BloomFilterMightContain`, but users cannot use it as predicate directly.

Reply:
Yes.